components/NavigationBar.jsx

```diff
@@ -92,7 +92,9 @@ export default class NavigationBar extends React.Component {
                     options={applicationUrls.map((option) => {
                       return {
                         ...option,
-                        title: <span><b>Caseflow</b> {option.title}</span>
+                        title: option?.link === '/decision_reviews/vha' ?
```

Comment:
```suggestion
                        title: <span><b>{option.prefix || Caseflow}</b> {option.title}</span>
```
I don't think this should be hardcoded to a specific url. What if you added another key to the option hash named something like prefix?  That way it could be expanded in the future to more than just this single link url